src/types.d.ts

```diff
@@ -58,6 +58,7 @@ export type RuleViolation = {
 	endColumn?: number;
 	ruleName: string;
 	severity: number;
+	normalizedSeverity?: number;
```

Comment:
this optional field has been added for when the normalizer is used, the value is undefined when run without -n

Reply:
Is the long-term plan to eventually replace the old severity functionality entirely, and rename this to just `severity`?

Reply:
Based on our last parking lot discussion, both would probably stay.